### PR TITLE
feat: disable actions of unmanaged experiments/trials

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1631]

## Test Plan
- Hide `Profiler` tab
- Disable actions of unmanaged experiment/trial 
  - Hyperparameter Search button in `Hyperparameter` tab
  - Fork 
  - Continue Trial 
  - View in Tensorboard
  - Experiment List 
    - Pause 
    - Kill 
    - Cancel 
    - Resume 
    - View in Tensorboard 
 - For Tensorboard bulk action
   - Show Modal if unmanaged experiments are included

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1631]: https://hpe-aiatscale.atlassian.net/browse/WEB-1631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx [hunk 1]
```diff
@@ -249,8 +251,13 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
...
           <TrialDetailsHyperparameters pageRef={pageRef} trial={trialDetails as TrialDetails} />
         ),
+        disabled: experiment.unmanaged,
         key: TabType.Hyperparameters,
-        label: 'Hyperparameters',
+        label: experiment.unmanaged ? (
+          <Tooltip content={UNMANAGED_MESSAGE}>Hyperparameters</Tooltip>
+        ) : (
+          'Hyperparameters'
+        ),
```
Comment (gt2345) [inline webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx:260]: I don't think we want to hide the entire `Hyperparameters` tab, only `Hyperparameters Search` is not supported

File: webui/react/src/components/ExperimentTensorBoardModal.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  workspaceId?: number;
+}
+
+const ExperimentTensorBoardModal = ({
+  workspaceId,
+  experimentIds,
+  filters,
+}: Props): JSX.Element => {
+  const handleSubmit = async () => {
+    openCommandResponse(await openOrCreateTensorBoard({ experimentIds, filters, workspaceId }));
```
[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/components/ExperimentTensorBoardModal.tsx:19]: The backend doesn't filter it? If so, i'll change it

File: webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx:260]: ok. I misunderstood the ticket description.  Will fix it

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx [hunk 3]
```diff
@@ -414,6 +427,11 @@ const TableActionBar: React.FC<Props> = ({
         sourceWorkspaceId={project.workspaceId}
         onSubmit={handleSubmitMove}
       />
+      <ExperimentTensorBoardModalComponent
+        experimentIds={Array.from(selectedExperimentIds)}
+        filters={selectAll ? filters : undefined}
+        workspaceId={project?.workspaceId}
+      />
```
[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:434]: agreed. will fix it

**Comment by keita-determined** [comment]
@gt2345  fixes are pushed

File: webui/react/src/constant.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const UNMANAGED_MESSAGE = 'Not available for unmanaged workloads';
+
+export const UNMANAGED_EXPERIMENT_ANNOTATION_MESSAGE =
+  'Unmanaged experiments are selected, however, those experiments will be ignored';
```
Comment (gt2345) [inline webui/react/src/constant.ts:4]: nit: add . at the end

**Review by gt2345 — APPROVED** [review]
LGTM

File: webui/react/src/constant.ts (hunk 4 above)
Comment (keita-determined) [reply webui/react/src/constant.ts]: done

**Pull request merged**